playground/src/components/storybook/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import buttonReadme from '../../../../packages/components/src/button/README.md';
+import buttonGroupReadme from '../../../../packages/components/src/button-group/README.md';
+
+import checkboxControlReadme from '../../../../packages/components/src/checkbox-control/README.md';
+import radioControlReadme from '../../../../packages/components/src/radio-control/README.md';
+import rangeControlReadme from '../../../../packages/components/src/range-control/README.md';
+import selectControlReadme from '../../../../packages/components/src/select-control/README.md';
+import textControlReadme from '../../../../packages/components/src/text-control/README.md';
+import toggleControlReadme from '../../../../packages/components/src/toggle-control/README.md';
+
+import dropdownReadme from '../../../../packages/components/src/dropdown/README.md';
+import dropdownMenuReadme from '../../../../packages/components/src/dropdown-menu/README.md';
```

Comment:
I see an error in my Terminal when running:

>[1] 🚨  /Users/andrew/Documents/Code/vvv/www/editor/htdocs/wp-content/plugins/gutenberg/playground/src/packages/components/src/dropdown-menu/README.md: ENOENT: no such file or directory, open '/Users/andrew/Documents/Code/vvv/www/editor/htdocs/wp-content/plugins/gutenberg/playground/src/packages/components/src/dropdown-menu/README.md'
[1] Error: ENOENT: no such file or directory, open '/Users/andrew/Documents/Code/vvv/www/editor/htdocs/wp-content/plugins/gutenberg/playground/src/packages/components/src/dropdown-menu/README.md'

It seems to be navigating `packages` relative to `playground/src` than to the root?

Reply:
I'm seeing a similar issue now, weird, I don't know what's happening :)

Reply:
So it looks like parcel tries to read the links (dependencies) fo the `menu-item` README file and it's pointing to the `drop-down` menu using an absolute link `/something` which break this. I temporarily commented that README (the menu item) but I guess this means we need another way to load the content of these markdown files. I was actually wondering if using `markdown-it` would be better longer term.